tests/UI/campaigns/functional/FO/09_productPage/03_writeReview.js

```diff
@@ -0,0 +1,356 @@
+
+  after(async () => {
+    await helper.closeBrowserContext(browserContext);
+  });
+
+  describe('Go to FO product detail page and add a review', async () => {
+    it('should open the shop page', async function () {
+      await testContext.addContextItem(this, 'testIdentifier', 'openFoShop', baseContext);
+
+      await foHomePage.goTo(page, global.FO.URL);
+
+      const result = await foHomePage.isHomePage(page);
+      await expect(result).to.be.true;
+    });
+
+    it('should go to login page', async function () {
+      await testContext.addContextItem(this, 'testIdentifier', 'goToFOLoginPage', baseContext);
+
+      await foHomePage.goToLoginPage(page);
+
+      const pageHeaderTitle = await foLoginPage.getPageTitle(page);
+      await expect(pageHeaderTitle).to.equal(foLoginPage.pageTitle);
+    });
+
+    it('should login with default customer', async function () {
+      await testContext.addContextItem(this, 'testIdentifier', 'signInFo', baseContext);
+
+      await foLoginPage.customerLogin(page, DefaultCustomer);
+
+      const isCustomerConnected = await foLoginPage.isCustomerConnected(page);
+      await expect(isCustomerConnected, 'Customer is not connected').to.be.true;
+    });
+
+    it('should go to home page', async function () {
+      await testContext.addContextItem(this, 'testIdentifier', 'goToHomePageToAddReview', baseContext);
+
+      await foHomePage.goToHomePage(page);
+
+      const result = await foHomePage.isHomePage(page);
+      await expect(result).to.be.true;
+    });
+
+    it('should go to the first product page', async function () {
+      await testContext.addContextItem(this, 'testIdentifier', 'goToProductPage', baseContext);
+
+      await foHomePage.goToProductPage(page, 1);
+
+      const pageTitle = await productPage.getPageTitle(page);
+      await expect(pageTitle.toUpperCase()).to.contains(ProductData.firstProductData.name);
+    });
+
+    it('should get and save the actual comment count', async function () {
+      await testContext.addContextItem(this, 'testIdentifier', 'getFoCommentCount', baseContext);
+
+      foCommentCount = await productPage.getNumberOfComment(page);
+      await expect(foCommentCount).to.be.at.least(0);
+    });
+
+    it('should add a product review', async function () {
+      await testContext.addContextItem(this, 'testIdentifier', 'addProductReview', baseContext);
+
+      const isReviewSent = await productPage.addProductReview(page, productReviewData);
+      await expect(isReviewSent).to.be.true;
+    });
+
+    it('should sign out from FO', async function () {
+      await testContext.addContextItem(this, 'testIdentifier', 'signOutFO', baseContext);
+
+      await productPage.logout(page);
+      const isCustomerConnected = await productPage.isCustomerConnected(page);
+      await expect(isCustomerConnected, 'Customer is connected').to.be.false;
+    });
+  });
+
+  describe('Go to BO and go to "product comments" module configuration page', async () => {
```

Comment:
```suggestion
  describe('Go to BO and go to \'product comments\' module configuration page', async () => {
```

Please do use `"` and use `'` instead